docs/search.md

```diff
@@ -37,6 +37,63 @@ curl -XGET 'https://…/api/v1/search' -d '{
 }'
 ```
 
+## Advanced Search 
+
+Advanced Search is based on [Elasticsearch](https://www.elastic.co/). It comprises of spatio-temporal search and keyword search.
+
+### Example requests
+
+- [http://o2r.uni-muenster.de/api/v1/search?q=*&from&to&coords&keyword](http://o2r.uni-muenster.de/api/v1/search?q=*&from&to&coords&keyword)
```

Comment:
This example must have actual values, e.g.

```
http://o2r.uni-muenster.de/api/v1/search?q=*&from=2017...&to=2017...&coords=<how the coords look like>&keyword=example_keyword1
```

Just like the one below, but maybe without the coordinates to keep it simple.